examples/rcn/load_data.py

```diff
@@ -0,0 +1,78 @@
+# export
+import os
+
+import cv2
```

Comment:
I don't think it's worth it to introduce an additional dependency just for resizing....You can use https://jax.readthedocs.io/en/latest/jax.image.html instead